config/application.rb

```diff
@@ -1,10 +1,13 @@
 require_relative 'boot'
 
 require 'rails/all'
-
+require 'assembly-utils'
```

Comment:
does this `require 'assembly-utils` statement in `config/application.rb` mean the corresponding one in `lib/pre-assembly.rb` can be removed?